lib/steps/load_state.go

```diff
@@ -5,18 +5,41 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
 )
 
-func loadState(command string) RunState {
+func hasSavedState(command string) bool {
 	filename := getRunResultFilename(command)
-	content, err := ioutil.ReadFile(filename)
+	_, err := os.Stat(filename)
 	if err != nil {
-		log.Fatal(err)
+		if os.IsNotExist(err) {
```

Comment:
Oh. Okay, please add the return at the end. `golint` does not like if-else clauses that all contain `return` statements. It suggests to remove the last `else` statement and unindent the block inside it.